api/package.json

```diff
@@ -5,11 +5,10 @@
   "description": "API for the vercel/vercel repo",
   "main": "index.js",
   "scripts": {
-    "vercel-build": "node ../utils/run.js build all"
+    "vercel-build": "yarn --cwd .. build"
```

Comment:
Do we still need this API? I thought this PR was making all the endpoints static?